native/src/components/CalendarChoiceModal.tsx

```diff
@@ -33,15 +66,49 @@ const CalendarChoiceModal = ({
   modalVisible,
   closeModal,
   eventTitle,
+  recurring,
 }: CalendarChoiceProps): ReactElement => {
   const { t } = useTranslation('events')
+  const [selectedCalendar, selectCalendar] = useState(calendars[0]?.id ?? '')
```

Comment:
here is a type missing for the state. i suggest to set null as fallback instead of empty string